docs/remote_inputs.md

```diff
@@ -0,0 +1,156 @@
+# Overview of remote nCoV files (intermediate build assets)
+
+We have two repos which routinely upload files to s3 buckets: `ncov-ingest` and `ncov`.
+Each of those runs separate pipelines for `gisaid` and `genbank` (aka `open`) data sources, with the intention that the latter are available for public download.
+These files are intended to be available for use as starting points for running the `ncov` pipeline.
+This page details where different assets come from, and how they may be used as inputs for your own workflow.
+
+The GISAID data is stored at `s3://nextstrain-ncov-private` and the open (GenBank) data on the public bucket `s3://nextstrain-data/files/ncov/open`
+The latter is intentionally made public in the hope that others can use this data as inputs for their workflows (see below).
+
+## All available genomes and metadata
+Entire metadata & sequences data is uploaded from the `ncov-ingest` workflows for each of the `gisaid` and `open` sources:
+
+* `metadata.tsv.gz`
+* `sequences.fasta.gz`
+* `nextclade.tsv.gz`
+* `additional_info.tsv.gz` (GISAID only)
+* `flagged_metadata.txt.gz` (GISAID only)
+
+The `ncov` repo contains two nextstrain-only workflows which use the above data to generate our analyses visible on nextstrain.org.
+A side-effect of this is the creation and upload of processed versions of the entire dataset:
+
+* `aligned.fasta.xz` alignment via [nextalign](https://github.com/nextstrain/nextclade). The default reference genome is MN908947 (Wuhan-Hu-1).
+* `mutation-summary.tsv.xz` A summary of the data in `aligned.fasta.xz`.
+* `masked.fasta.xz` Masked alignment
+* `filtered.fasta.xz` The masked alignment with sequences excluded due to incomplete / invalid dates, unexpected genome length, missing metadata. We also maintain a [list of sequences to exclude](https://github.com/nextstrain/ncov/blob/master/defaults/exclude.txt).
```

Comment:
It might be worth clarifying here if this `filtered` file has the sequences from `exclude` already excluded, or not - for me reading this I feel it's ambiguous!

Reply:
Done, thanks!